test/zdtm/static/cgroup_threads.hook

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+[ "$1" == "--clean" -o "$1" == "--pre-restore" ] || exit 0
+
+set -e
+
+tname=$(mktemp -d cgclean.XXXXXX)
+mount -t cgroup none $tname -o "none,name=zdtmtst"
+
+echo "Cleaning $tname"
+
+set +e
+rmdir "$tname/subcg_threads/subsubcg/"
+rmdir "$tname/subcg_threads/"
+set -e
+
+echo "Left there is:"
+ls "$tname"
+umount "$tname"
+rmdir "$tname"
```

Comment:
With the `trap`s added, those are not needed.

Reply:
Done.